modules/openapi-generator/src/main/resources/dart2/api_helper.mustache

```diff
@@ -54,7 +57,7 @@ String parameterToString(dynamic value) {
 /// Returns the decoded body by utf-8 if application/json with the given headers.
 /// Else, returns the decoded body by default algorithm of dart:http.
 /// Because avoid to text garbling when header only contains "application/json" without "; charset=utf-8".
-String _decodeBodyBytes(Response response) {
+String decodeBodyBytes(Response response) {
```

Comment:
I notice that you removed the leading underscore so these methods are now officially public, right?

Reply:
Yeah they've gone from being "library private" to "library public".  Ironically, this has made them less 'public' - because the package was using the "part of" directive, the whole package was effectively a single library unit meaning the methods were accessible anywhere.  Now the api_helper.dart has to be imported before they can be used.  Externally there is no change as the api_helper.dart file is not exported.

Reply:
Thanks for the explanation.